### ISSUE TITLE
Impossible to clear indices via CLI

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch-symfony#12]
- I ran a local MS instance with an index named `indexUID` containing books with a `title`
- I created a new Symfony project
- I added:
```bash
MEILISEARCH_URL=http://127.0.0.1:7700
MEILISEARCH_API_KEY=masterKey
```
in my project
- I added this file (`/config/package/meili_search.yaml`) in my project:
```yml
meili_search:
     indices:
          -   name: indexUID
               class: App\Entity\Book
               enable_serializer_groups: true
```
(I've also created an `src/Entity/Book.php` file following this [part of the docs](https://github.com/meilisearch/meilisearch-symfony/wiki/configure-searchable-data#using-annotations))
- Following [this part of the docs](https://github.com/meilisearch/meilisearch-symfony/wiki/index-data-into-meilisearch#removing-manually), I did:
```bash
$ php bin/console meili:clear
```
And I got:
```bash
Index indexUID  couldn't be cleared
Done!
```

The index was indeed not removed.
Plus, an empty `dev_indexUID` was created in my MeiliSearch instance.

Versions:
- MeiliSearch `v0.12.0`
- meilisearch/search-bundle `v0.2.0`
- Symfony `v4.17.0`

**Comment by emulienfou** [comment]
@curquiza I think you have this error because of the default value of `MEILISEARCH_PREFIX` who take the environment name.
You should create an index `dev_indexUID` or set the value of `MEILISEARCH_PREFIX` to an empty value.

Maybe the documentation is not clear about that

**Comment by curquiza** [comment]
Oh I indeed did not read [this part of the docs](https://github.com/meilisearch/meilisearch-symfony/wiki/index-configuration#multi-environment-setup) because it was for multi environment setup and I thought I wasn't because I just wanted to test.
(That's the first time I'm using Symfony to be honest, so I'm not the best user in the world 😅)

I succeed to clear my index 🎉 but there is still a little issue: despite everything is cleared, the prompt still indicates:
```bash
Index indexUID  couldn't be cleared
Done!
```

**Comment by curquiza** [comment]
closed by #13
